contentcuration/contentcuration/frontend/channelEdit/components/ContentNodeValidator.vue

```diff
@@ -75,16 +75,13 @@
           : '';
       },
       error() {
-        if (!this.node.complete) {
```

Comment:
It’s a bit unclear to me why this check is being removed. It appears to validate whether the node is incomplete. I think our investigation should focus on understanding why the node becomes marked as incomplete after updating the title (and possibly other fields). Removing this validation check would only mask the issue and prevent us from detecting when a node is actually incomplete. Am I missing something?